service/dap/server.go

```diff
@@ -788,6 +836,7 @@ func (s *Server) doContinue() {
 		}
 		return
 	}
+	s.stackFrameHandles.reset()
```

Comment:
I may be missing something here, but where to you actually use `stackFrameHandles`? The code seems to create unique handles and reset the handles. I don't see `.get` calls anywhere

Reply:
I am not using them in this PR. But I will be using them in the next PR, which cross-references things as part of scopes and variables requests. For now it is here to make sure all the logs match those from the existing adaptor just for this particular request. When I mail this out to delve, I can add a comment in the review for them explaining this, so they are not wondering the same.